ipt_connect/IPTdev/views.py

```diff
@@ -387,9 +387,17 @@ def physics_fight_detail(request, pfid):
 				pass
 			meanroundsgrades.append(meangrades)
 
-		for team in summary_grades:
-			summary_grades[team].append(sum(summary_grades[team]))
-		summary_grades = sorted(summary_grades.items(), key=lambda x: x[1][4], reverse=True)
+		try:
+			for team in summary_grades:
+				summary_grades[team].append(sum(summary_grades[team]))
+			summary_grades = sorted(summary_grades.items(), key=lambda x: x[1][4], reverse=True)
+		except:
```

Comment:
In which cases would the above fail?

Reply:
`x[1][4]` can be undefined, e.g. for 4-fight.